plugins/hypervisors/kvm/src/main/java/org/apache/cloudstack/utils/qemu/QemuImg.java

```diff
@@ -149,6 +149,8 @@ public void create(final QemuImgFile file, final QemuImgFile backingFile, final
          */
         s.add("-f");
         if (backingFile != null) {
+            s.add(backingFile.getFormat().toString());
+            s.add("-F");
```

Comment:
@weizhouapache I checked libvirtd release notes, libvirt doesn't fail VM for version 5.x or below; only libvirt starting version 6.x fails VM deployment if backing file format is not provided. Should we add a check, as the `-F` option is not available for qemu-img on older distros, say CentOS7, or Ubuntu 16.04 (I didn't check 18.04). Unless we want to discuss and deprecate Ubuntu 16.04 support? cc @wido @GabrielBrascher @shwstppr @davidjumani

Reply:
@rhtyd 
I think it is supported by ubuntu 16.04
I have tested the migration from ubuntu 16.04 to ubuntu 20.04.
it works fine.

not sure if centos7 supports it.

Reply:
@weizhouapache I'm cherry-picking couple of your commits in centos8 support PR branch for testing. I hope it is fine with you.

Reply:
@shwstppr while you do the tests etc, can you check if qemu-img on CentOS7 supports the `-F` argument for specifying the backfile file format?